src/shared/api/schemas.ts

```diff
@@ -21,12 +21,10 @@ export interface Paginated<T> {
   items: T[]
 }
 
-const PaginationParams = z
-  .object({
-    page: z.number(),
-    per_page: z.number(),
-  })
-  .partial();
+const PaginationParams = z.object({
+  page: z.number(),
+  per_page: z.number(),
+});
```

Comment:
I see that this schema isn't used anywhere. Therefore, it is enough to define it just as an `interface`.